backend/src/auth/signup.strategy.ts

```diff
@@ -25,7 +25,24 @@ export class SignupStrategy extends PassportStrategy(Strategy, 'signup') {
     signupDto.password = req.body.password;
     const errors = await validate(signupDto);
     if (errors.length > 0) {
-      throw new BadRequestException();
+      console.log(errors[0].constraints.matches)
```

Comment:
Should remove this line except if specific reason

Reply:
done